script/seccomp.sh

```diff
@@ -19,8 +24,12 @@ function build_libseccomp() {
 	local arches=("$@")
 	local tar="libseccomp-${ver}.tar.gz"
 
-	# Download and extract.
+	# Download, check, and extract.
 	wget "https://github.com/seccomp/libseccomp/releases/download/v${ver}/${tar}"{,.asc}
+	if ! sha256sum "${tar}" | grep -E "^${SECCOMP_SHA256[${ver}]}\s+${tar}$"; then
```

Comment:
Not a maintainer (so take this for what it's worth), but given this is Bash, I think this would be more reliable something like this:

```suggestion
	if sha256sum --strict --check - <<<"${SECCOMP_SHA256[${ver}]} *${tar}"; then
```

Reply:
Yeah I agree that's more readable.